tutorials/CleanRL/requirements.txt

```diff
@@ -1,5 +1,5 @@
 SuperSuit==3.7.2
 torch==1.13.1
-pettingzoo[butterfly]==1.22.4
```

Comment:
I think these three files need this line to be fixed? Maybe you just need to solve the merge conflict

Reply:
Good catch I’ll update it when I get home

Reply:
Wait I don’t think this needs to be changed, 1.22.4 got yanked so it needs to be 1.22.3

Reply:
Oh I see there was a merge conflict, got it figured out